common/persistence/visibility/store/query/converter.go

```diff
@@ -35,6 +35,7 @@ import (
 
 	"github.com/olivere/elastic/v7"
 	"github.com/xwb1989/sqlparser"
+	"go.temporal.io/server/common/persistence/visibility/store/elasticsearch/client"
```

Comment:
This is why `client.SearchParameters` were not used here. `client.SearchParamters` are part of the contract between store and client but not between query converter and store. Can you define another struct here, return it to store, and then copy values to `client.SearchParamters` in the store.

Reply:
Create another struct `QueryParams` in the query converter.